axum/src/docs/method_routing/route_layer.md

```diff
@@ -19,10 +20,9 @@ use axum::{
 };
 use tower_http::validate_request::ValidateRequestHeaderLayer;
 
-let app = Router::new().route(
-    "/foo",
-    get(|| async {})
-        .route_layer(ValidateRequestHeaderLayer::bearer("password"))
+let app = Router::new()
+    .route("/foo", get(|| async {})
+    .route_layer(ValidateRequestHeaderLayer::bearer("password"))
```

Comment:
This change calls `Router::route_layer` but these docs are for `MethodRouter::route_layer`. So the code was internally like the way it was before.

Reply:
ah sorry, I didn't mention this. Yes, that line did not change, I only "formatted" it to make it look like the many other code examples in the documentation. 
I only meant the wording in the description/text of the documentation itself where it says `.route` but I think it should say `.route_layer` 

If you want I can remove that "formatting" change from that commit or put it in its own commit

Reply:
Oh, now I see what you mean. I didn't notice this actually. I guess it would be nice to have a correct example where `MethodRouter::route_layer` is called :sweat_smile:

Reply:
nevermind, I was confused. So the "formatting" change does not change the functionality. It still calls the `route_layer` on the `.get(..)` method, however, the new formatting makes that less obvious which is bad.

Reply:
already done :sweat_smile: